src/shared/Button/Button.scss

```diff
@@ -0,0 +1,26 @@
+button{
+    border: none;
+    padding: 10px 35px;
+    border-radius: 50px;
+    background-color: #eeee22;
```

Comment:
you can create `coros.scss` file, and declare variables there for each color.
Then import that file in `Button.scss` and use color from variable.
This way it would be easier to adjust colors later, if needed.